File: exercises/small_problems_101/easy_2/how_big_is_the_room.rb
SQUARE_METERS_TO_FEET = 10.7639

puts "Enter the length of the room in meters:"
length = gets.chomp.to_f
puts "Enter the width of the room in meters:"
width = gets.chomp.to_f

area_meters = length * width
area_feet = length * width * SQUARE_METERS_TO_FEET

puts "The area of the room is #{area_meters} meters (#{area_feet.round(2)}) sqaure feet"

File: frontend/javascript/exercises/easy_1/how_big_is_the_room.js
var CONVERSION = 10.7639;
var length = window.prompt('Enter the length of the room in meters:');
var width = window.prompt('Enter the width of the room in meters:');
var area = Number(length) * Number(width);
var convertedArea = area * CONVERSION;
console.log('The area of the room is ' + area.toFixed(1) + ' square meters ' + '(' + convertedArea.toFixed(2) + ' square feet).');